### PR TITLE
Test Tools (God Mode)

### PR DESCRIPTION
**Pull request by AlbertVVila**
# Les Features
- **F1 - F2 for switching to either level 1 or 2**
- **F3 for adding 100 points of experience**
- **F4 toggles infinite health**
- **F5 toggles infinite mana**
- **F6 toggles boosts in stats (insta kills + super speed)**
- **F7 - F8 to increase and decrease brightness**
- Added in _Tools_ section a button to delete all serialized data

# Le Test
- Check that these features work as described in the previous section.
- In order to check the delete button in _Tools_ section you can load scene 2 and check that, when you click the delete button, the content of the persistence folder is deleted.

# Le Bug
Detected a bug in persistence. When you change scene, the inventory and player information is saved but it is only deleted when you click in _New Game_. We should delete it on STOP.

### CODE REVIEW

File: FractalEngine.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2042
+# Visual Studio Version 16
```
Comment (Rubetoman) [inline FractalEngine.sln]: Couldn't this lead to errors for people using the previous version?

File: Scripts/GodMode/GodMode.cpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+void GodMode::ClickedGodMode()
+{
+	if (App->input->GetKey(SDL_SCANCODE_G) != KEY_DOWN) return;
+	isGod = !isGod;
+}
+
+void GodMode::SwitchLevel() const
+{
+	unsigned level = 1;
```
Comment (Rubetoman) [inline Scripts/GodMode/GodMode.cpp]: 1u

File: Scripts/GodMode/GodMode.cpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	if (App->input->GetKey(SDL_SCANCODE_1) == KEY_DOWN)
+	{
+		level = 1;
```
Comment (Rubetoman) [inline Scripts/GodMode/GodMode.cpp]: 1u

File: Scripts/GodMode/GodMode.cpp [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	}
+	else if (App->input->GetKey(SDL_SCANCODE_2) == KEY_DOWN)
+	{
+		level = 2;
```
Comment (Rubetoman) [inline Scripts/GodMode/GodMode.cpp]: 2u

File: FractalEngine.sln (hunk 1 above)
Comment (AlbertVVila) [reply FractalEngine.sln]: This only updates the solution version. The compatibility version is this one : _MinimumVisualStudioVersion_ and it has not changed.

**Comment by AlbertVVila** [comment]
Corrected typos + Forgot to upload the GodMode prefab and scene modifications. Tomorrow i will update the description and test methodology

**Comment by AlbertVVila** [comment]
Look at description for updated version!! :D

**Comment by AlbertVVila** [comment]
Increased brightness speed and updated HUD on modifying player stats. 👍

**Comment by solidajenjo** [comment]
Keep a minimum of exposure for god's sake!!

**Comment by AlbertVVila** [comment]
Bounded exposure 😃

**Pull request merged**